packages/venia-ui/lib/components/Gallery/item.css

```diff
@@ -69,3 +70,51 @@
     width: 100%;
     font-size: var(--venia-global-fontSize-100);
 }
+
+.ratingAverage {
+    position: absolute;
+    bottom: 0.75rem;
+    right: 0.75rem;
+    display: grid;
+    grid-template-columns: auto auto;
+    padding: 0.5rem;
+    align-items: center;
+    column-gap: 0.5rem;
+    background-color: rgb(var(--venia-global-color-gray-darker));
+    color: rgb(var(--venia-global-color-gray-50));
+}
+
+.ratingValue {
+    font-weight: var(--venia-global-fontWeight-semibold);
+}
+
+.ratingIcon {
+    composes: root from '../Icon/icon.css';
+    --stroke: rgb(var(--venia-global-color-gray-50));
+}
+
+/* state: pending */
```

Comment:
The pending classes were removed in v12. Is there a reason for adding them back in?

Reply:
@JustinConabreeAbsolunet I just want to style color gray for icon. Cause removed on v12, can you give me a hex color to fill this?

Reply:
@huykon The styles you moved to `rating.css` are all good. It's just the `*_pending` classes that are in this file that are no longer necessary. They were there to show a loading state of the item, but that logic has now been moved into `packages/venia-ui/lib/components/Gallery/item.shimmer.js`